.ats/main.yaml

```diff
@@ -0,0 +1,9 @@
+app-tests-app-config-file: tests/test-values.yaml
+
+skip-steps: functional
+
+smoke-tests-cluster-type: kind
+
+upgrade-tests-cluster-type: kind
+upgrade-tests-app-catalog-url: https://giantswarm.github.io/giantswarm-catalog
+upgrade-tests-app-config-file: tests/test-values.yaml
```

Comment:
Why is this not used for smoke tests?

Reply:
Not sure what you mean. it defines kind for smoke test clusters in line 5, or do you mean something else?

Reply:
the `upgrade-tests-app-config-file: tests/test-values.yaml` line. Why is the test config file not used for smoke tests? Its possible to specify analogue to this using `smoke-tests-cluster-config-file`

Reply:
Scratch that, its already good like it is